AppCenterIdentity/AppCenterIdentity/MSIdentityErrors.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenterErrors.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+#pragma mark - Domain
+
+/**
+ * Error domain for Identity.
+ */
+static NSString *const kMSIdentityErrorDomain = MS_APP_CENTER_BASE_DOMAIN @"Identity.ErrorDomain";
+
+/**
+ * Error description key for Identity.
+ */
+static NSString *const kMSIdentityErrorDescriptionKey = @"MSIdentityErrorDescriptionKey";
```

Comment:
nit: why is "Key" In the actual name of the key? I recommend `@"MSIdentityErrorDescription"`

Reply:
(i know this is just moved code but I think this is a good time to change that)

Reply:
This is an actual key used in userInfo. We have Key suffix for a key used in userInfo dictionary.